kernel/src/process/program_loader/elf/load_elf.rs

```diff
@@ -49,10 +48,12 @@ pub fn load_elf_to_vm(
     let ldso = lookup_and_parse_ldso(&elf_headers, &elf_file, fs_resolver)?;
 
     match init_and_map_vmos(process_vm, ldso, &elf_headers, &elf_file) {
+        #[cfg_attr(target_arch = "loongarch64", expect(unused_mut))]
         Ok((_range, entry_point, mut aux_vec)) => {
             // Map and set vdso entry.
             // Since vdso does not require being mapped to any specific address,
             // vdso is mapped after the elf file, heap and stack are mapped.
```

Comment:
Sorry. I missed this in #2362. Let's fix it here.
```suggestion
            // Map the vDSO and set the entry.
            // Since the vDSO does not require being mapped to any specific address,
            // the vDSO is mapped after the ELF file, heap, and stack.
```